src/vsr/journal.zig

```diff
@@ -613,7 +583,6 @@ pub fn JournalType(comptime Replica: type, comptime Storage: type) type {
             op_max: u64,
         ) ?HeaderRange {
             assert(journal.status == .recovered);
-            assert(journal.header_with_op(op_min) != null);
```

Comment:
We used to require that `op_min` exists, but the replica can't make that guarantee.